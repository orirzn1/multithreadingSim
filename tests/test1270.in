L12
S6
O7
N6

T0
I@0x00000000
ADDI $1, $4, 0x1
SUB $1, $1, $7
LOAD $7, $7, 0x4
STORE $7, $5, $2
SUBI $4, $2, 0x2
SUBI $0, $2, 0x2
HALT $0

T1
I@0x00000000
STORE $5, $1, $4
LOAD $7, $2, 0x0
ADDI $3, $3, 0x2
SUBI $6, $4, 0x4
STORE $7, $6, 0x2
ADD $3, $3, $4
HALT $0

T2
I@0x00000000
LOAD $7, $3, $2
SUB $2, $6, $3
SUBI $5, $1, 0x1
LOAD $3, $1, $0
ADD $5, $0, $7
ADDI $2, $3, 0x3
ADD $6, $5, $4
HALT $0

T3
I@0x00000000
ADD $0, $4, $7
SUB $5, $6, $5
LOAD $6, $4, $2
STORE $4, $1, 0x2
ADD $3, $3, $3
ADDI $4, $5, 0x2
HALT $0

T4
I@0x00000000
ADDI $1, $2, 0x1
STORE $7, $3, $0
LOAD $4, $1, 0x3
STORE $3, $4, 0x0
HALT $0

T5
I@0x00000000
STORE $0, $3, 0x1
STORE $1, $3, $3
LOAD $0, $2, 0x3
SUB $5, $4, $5
HALT $0

D@0x0
0xA
0xA
0x2
0x5
0x1
0x8
