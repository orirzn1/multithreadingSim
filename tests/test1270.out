
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0xFFFFFFFE	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0xA
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0xA	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0xA	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 4.589744

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0xFFFFFFFE	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0xA
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0xA	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.153846

